mailbag/parsers/randon.py

```diff
@@ -0,0 +1,62 @@
+from mailbag.parser import EmailFormatParser
+from mailbag.emailModel import Email
+
+import sys
+import io
+
+import pypff
+import pandas as pd
+
+
+
+
+class PST(EmailFormatParser):
+    """PST - This concrete class parses PST file format"""
+    parser_name = 'PST'
+
+
+
+
+
+
+    def parse(self,*args,**kwargs):
+
+        """Parses the PST file
+               Return:
+                    allMails (list) : list of email model object"""
+
+
+        allmails = []
+
+
+        for folder in self.sub_folders:
+            messages = []
+            mailObj = Email()
+            if folder.number_of_sub_folders:
+                messages += PST.parse(self=folder)
+                #print(messages)
+            #print(folder.name)
+            for msg in folder.sub_messages:
```

Comment:
Looks like we want `.sub_items` instead of `.sub_messages` here, which give better access to message objects.

There's not a lot of docs for libpff, I basically just tried `dir(folder)` and `dir(msg)` to see what the possible methods and attributes were. These attributes seem to work for msg objects if you use `.sub_items`: 

```
conversation_topic
get_client_submit_time
get_client_submit_time_as_integer
get_conversation_topic
get_creation_time
get_creation_time_as_integer
get_delivery_time
get_delivery_time_as_integer
get_html_body
get_identifier
get_modification_time
get_modification_time_as_integer
get_number_of_attachments
get_number_of_entries
get_number_of_record_sets
get_number_of_sub_items
get_plain_text_body
get_record_set
get_rtf_body
get_sender_name
get_sub_item
get_subject
get_transport_headers
html_body
identifier
number_of_attachments
number_of_entries
number_of_record_sets
number_of_sub_items
plain_text_body
record_sets
rtf_body
sender_name
sub_items
subject
transport_headers
```